lib/iris/cube.py

```diff
@@ -1117,19 +1117,36 @@ def remove_coord(self, coord):
         for factory in self.aux_factories:
             factory.update(coord)
 
-    def remove_cell_measure(self, cell_measure):
+    def remove_cell_measure(self, name_or_cell_measure):
```

Comment:
@alastair-gemmell I think that keeping the `cell_measure` positional argument "as is" is good enough, and is in keeping with other `remove_` methods that allow a mixture of different types of inputs to match on i.e., see `iris.cube.Cube.remove_coord` for example.

Reply:
Thanks @bjlittle .  I've made this change now and the tests have passed.

It's good to have this consistency with other `remove` methods, although there is already inconsistency because the `cell_measures` and `cell_measure` methods do have the argument listed as 'name_or_cell_measure' (which is what I based my original change on!)